lib/matplotlib/_color_data.py

```diff
@@ -961,14 +961,9 @@
     'green': '#15b01a',
     'purple': '#7e1e9c'}
 
-# normalize to names with no spaces and provide versions with XKCD
-# prefix.
-for k in list(XKCD_COLORS):
-    XKCD_COLORS['xkcd'+k] = XKCD_COLORS[k]
-    _k = k.replace(' ', '')
-    if _k != k:
-        XKCD_COLORS[_k] = XKCD_COLORS[k]
-        XKCD_COLORS['xkcd'+_k] = XKCD_COLORS[k]
+
+# Normalize name to "xkcd:<name>" to avoid name collisions.
+XKCD_COLORS = {'xkcd:' + name: value for name, value in XKCD_COLORS.items()}
```

Comment:
Can we then just standardize on the space-less versions then?  Having both just feels like... not being able to make up your mind.  (This would also solve the missing overlaps you mention below.)